### ISSUE TITLE
Prevent updating files if `gettext` shows a message

### ISSUE THREAD

**Issue by dalthviz** [spyder-ide/gettext-helpers#2]
As an example:

```
..spyder/spyder/plugins/projects/widgets/projectdialog.py:74: Seen unexpected token "f"Select a new or existing directory to create a new Spyder ""
```

Since the message could indicate an actual error. See https://github.com/spyder-ide/spyder/pull/16855

**Comment by ccordoba12** [comment]
@dalthviz, please take care of this one after we release 5.2.0. There's no rush to solve it, but it'd be nice to have a solution for it.

**Comment by CAM-Gerlach** [comment]
It seems the underlying issue here is that support f-strings are not supported, but not sure if there's anything we can do about it here or it has to be solved lower down in the dep stack

**Comment by ccordoba12** [comment]
The idea here is not allow updating the po files if gettext reports any message. Those messages look like warnings (because we don't exit on them), but they are really errors because the generated po files are unusable.

**Comment by CAM-Gerlach** [comment]
Right, I understand; I was suggesting opening another issue about supporting f-strings for `gettext`, but thinking about it some more, it isn't really possible (at least without hacks/helper functions). Sorry for the distraction.
